src/lifetimes/lifetime-annotations.md

```diff
@@ -12,12 +12,9 @@ also be explicit: `&'a Point`, `&'document str`. Lifetimes start with `'` and
 `'a` is a typical default name. Read `&'a Point` as "a borrowed `Point` which is
 valid for at least the lifetime `a`".
 
-Lifetimes are always inferred by the compiler: you cannot assign a lifetime
-yourself. Explicit lifetime annotations create constraints where there is
-ambiguity; the compiler verifies that there is a valid solution.
-
-Lifetimes become more complicated when considering passing values to and
-returning values from functions.
```

Comment:
I think this removed text is load-bearing in that it counteracts a common misconception that lifetimes annotations can alter the runtime behavior of programs (in terms of how long values exist and when they are destroyed). We don't need to keep this exact phrasing but in addition to describing lifetimes as constraints we should have some explicit indication that only ownership, not lifetime annotations, control when objects are destroyed and determine the concrete lifetime of a given value.

Reply:
Thanks! What do you think of the update?

Reply:
"The borrow checker just ensures this is done safely." implies that the borrow checker is validating ownership rather than borrowing; I think it would be better to say that "The borrow checker just validates that borrows never extend beyond the concrete lifetime of the value."

I also think it may be better to say "values" rather than "objects". While "objects" is perfectly correct, many students come from languages which make a distinction between objects and other kinds of values and may wrongly infer that the borrow checker only applies to non-primitives.